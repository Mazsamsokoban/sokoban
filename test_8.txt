loadtest 8
showstate w
showstate box
move w right
showmaze
showstate w
showstate box