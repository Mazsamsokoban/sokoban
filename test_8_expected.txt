f1:w f2:box f3:null 

name:w
field:f1
direction:Neutral
points:0
pushforce:10.0

name:box
field:f2
direction:Neutral
owner:null
pusher:null

f1:null f2:w f3:box 

name:w
field:f2
direction:Right
points:0
pushforce:10.0

name:box
field:f3
direction:Right
owner:null
pusher:w

